manifests/params.pp

```diff
@@ -31,6 +31,12 @@
     $service_enable = true
   }
 
+  if ($::operatingsystem == 'Ubuntu' and $::operatingsystemrelease in ['15.04','15.10','16.04','16.10']) or ($::operatingsystem == 'Debian' and $::operatingsystemrelease == '8') {
```

Comment:
Seems like i was a bit fast claiming 16.04 comes with 1.7.5... Not so sure if we enforce this facter version for one reason or another somewhere... 2.4.6 seems to be in 16.04.